utils/text_formatting.tsx

```diff
@@ -343,25 +337,8 @@ export function sanitizeHtml(text: string): string {
 }
 
 // Copied from our fork of commonmark.js
-const emailAlphaNumericChars = '\\p{L}\\p{Nd}';
-const emailSpecialCharacters = "!#$%&'*+\\-\\/=?^_`{|}~";
-const emailRestrictedSpecialCharacters = '\\s(),:;<>@\\[\\]';
-const emailValidCharacters = emailAlphaNumericChars + emailSpecialCharacters;
-const emailValidRestrictedCharacters = emailValidCharacters + emailRestrictedSpecialCharacters;
-const emailStartPattern =
-  '(?:[' +
-  emailValidCharacters +
-  '](?:[' +
-  emailValidCharacters +
-  ']|\\.(?!\\.|@))*|\\"[' +
-  emailValidRestrictedCharacters +
-  '.]+\\")@';
-const reEmail = XRegExp.cache(
-    '(^|[^\\pL\\d])(' +
-    emailStartPattern +
-    '[\\pL\\d.\\-]+[.]\\pL{2,4}(?=$|[^\\p{L}]))',
-    'g',
-);
+// eslint-disable-next-line no-useless-escape
```

Comment:
This eslint error as it complains for unnecessary escaping of `{` and `}` but with `u` flag it is required.